styleguide/derek/global-components/basement/basement.ejs

```diff
@@ -1,15 +1,24 @@
 <!--[basement]-->
 <div class="bg-dark-gray">
   <div class="basement-container">
-    <div class="col-sm-4">
+    <div class="col-sm-9">
       <div class="region-select">
         <span class="uppercase white basement-region-select">Change your region:</span>
         <a class="basement-flag-link" data-toggle="modal" id="region" data-target="#regionSelect">
           <div class="basement-flag-icon basement-flag-uk"></div>
         </a>
       </div>
+      <div class="basement-currency col-lg-pull-1">
```

Comment:
From a zoolander perspective, we need to be able to illustrate (and more importantly test) the basement both with and without a pricing selector. 

I'd recommend making this conditional using jade.
Basically the idea would be 
```
<% if(!hide_currency_selector) { var hide_currency_selector = false; } %>
```

And then wrap this code with something like this:
```
<% if (!hide_currency_selector) { %>
  Your code here
<% } %>
```

Then in references to the basement.ejs file, we set `hide_currency_selector` to `true`, except for one particular instance.